CONTRIBUTING.md

```diff
@@ -185,6 +185,8 @@ Then run `make` to build the binary. The binary in `_output` can be directly use
 
 You can run `make install` to make finch binary globally accessible.
 
+NOTE: If your build environment experiences issues with downloading dependencies from the GO Proxy Servers, then you may want to set the environment variable `export GOPROXY=direct`
```

Comment:
> GO Proxy Servers

Any particular reason for this capitalization?

Reply:
No, this was probably a typo, so I can remove the capitalization.

Reply:
Please see the new version ([comparison](https://github.com/runfinch/finch/compare/a30e73452c996599417c98608edaf38507b31389..74e9f8f133adcb3833d16b367035ca92fff5c730)), which removes the capitalization.